arm_controller:
  type: "position_controllers/JointTrajectoryController"
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
  gains:
    joint_1: {p: 100.0, d: 1.0, i: 0.0}
    joint_2: {p: 100.0, d: 1.0, i: 0.0}
    joint_3: {p: 80.0,  d: 1.0, i: 0.0}
    joint_4: {p: 50.0,  d: 0.5, i: 0.0}
    joint_5: {p: 30.0,  d: 0.3, i: 0.0}
